cfpb/cfgov-refresh
# Adjustments to rate checker css and layout

Adjustments to rate checker css and layout

## Additions
- Added the helpers.less file which is used by the rate checker. 

## Removals

- Removed unused handlebars templates. These templates were left over from a decommissioned OAH project.  @virginiacc  please verify. 

- Removed some unused less files from pages that are being migrated to Wagtail.


## Changes

- Modified `cfgov/unprocessed/apps/owning-a-home/webpack-config.js` to fix issues with requiring templates and the handlebars loader.  Also, updated the handlebars dependency to fix a bug introduced with the introduction of [Webpack 4](https://github.com/pcardune/handlebars-loader/pull/153).

- Modified css and html to fix issues with the layout.  I also modified some `calculator` breakpoints in conjunction with using a larger font size ( from 1000px to 1230px ).  There should be no other layout changes. 

## Testing

1. Run `frontend.sh`
2. Visit the rate checker and compare if to the production version.
3. Verify there are no errors in the console.

## Screenshots


## Notes

- At some point we will need to consolidate the rate checker break points.
- I used `https://github.com/ream88/stylelint-config-idiomatic-order` to order the properties so things my might be squirrly. 

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
